.github/workflows/gh-actions.yml

```diff
@@ -88,6 +96,7 @@ jobs:
     steps:
       - uses: actions/checkout@v2
       - run: $RUNNER --test262-esnext=update --test262-test-list=language
+      - run: $RUNNER --test262-esnext=update --test262-test-list=language --build-debug
```

Comment:
Debug artifact isn't uploaded, but it should.

Reply:
Didn't understand this statement.

Reply:
One line down you can see: "uses: actions/upload-artifact@v2" which upload build/tests/test262_tests_esnext/local/bin/test262.report as artifact to be able to download by the user and can analyze detailed test results. If we run debug tests inside the same job, we will need to upload those output too.